website/source/docs/providers/aws/r/api_gateway_authorizer.html.markdown

```diff
@@ -0,0 +1,49 @@
+---
```

Comment:
This docs page looks okay, although there's a missing link to this page which would have been added to `website/source/layouts/aws.erb`.